src/qt/forms/buynamespage.ui

```diff
@@ -153,5 +394,6 @@
  <resources>
   <include location="../bitcoin.qrc"/>
  </resources>
+ <resources/>
```

Comment:
This is an Electrum-ism, we already have a (non-empty) `resources` element (above).